stanford_media.install

```diff
@@ -57,3 +57,93 @@ function stanford_media_update_8003() {
   $display->setComponent('field_media_video_embed_field', $display_options);
   $display->save();
 }
+
+/**
+ * Move inline image alt text into a contrib supported attribute.
+ */
+function stanford_media_update_8004() {
+  foreach (_stanford_media_update_8004_get_filter_fields() as $field) {
+    list($entity_type, $field_name) = explode(':', $field);
+
+    try {
+      $entity_ids = \Drupal::entityQuery($entity_type)
+        ->condition($field_name, '%alt_text&quot%', 'LIKE')
+        ->execute();
+    }
+    catch (Exception $e) {
+      continue;
+    }
+
+    $entity_storage = \Drupal::entityTypeManager()->getStorage($entity_type);
+    foreach ($entity_storage->loadMultiple($entity_ids) as $entity) {
+      $field_values = $entity->get($field_name)->getValue();
+
+      foreach ($field_values as &$field_value) {
+        $field_value['value'] = _stanford_media_update_8004_fix_alt($field_value['value']);
+      }
+
+      $entity->set($field_name, $field_values);
+      $entity->save();
+    }
+  }
+}
+
+/**
+ * Get a list of all fields that are filtered.
+ *
+ * @return array
+ *   Array of values with entity type and field name.
+ */
+function _stanford_media_update_8004_get_filter_fields() {
+  $field_manager = \Drupal::service('entity_field.manager');
+  $filtered_text_fields = ['text', 'text_long', 'text_with_summary'];
+
+  $filtered_fields = [];
+  foreach ($field_manager->getFieldMap() as $entity_type => $fields) {
+    foreach ($fields as $field_name => $field_info) {
+      if (!in_array($field_info['type'], $filtered_text_fields)) {
+        continue;
+      }
+
+      $filtered_fields[] = "$entity_type:$field_name";
+    }
+  }
+  asort($filtered_fields);
+  return array_unique($filtered_fields);
+}
+
+/**
+ * Move the alt_text value in the json data into an `alt` attribute.
+ *
+ * @param string $html
+ *   Original Html with the embed entity tag.
+ *
+ * @return string
+ *   Fixed html.
+ */
+function _stanford_media_update_8004_fix_alt($html) {
+  preg_match_all("/<drupal-entity.*?\/drupal-entity>/s", $html, $tokens);
+  foreach ($tokens[0] as $token) {
+    // Existing tokens already use the contrib attribute or dont use an alt at
+    // all.
+    if (strpos($token, ' alt="') !== FALSE || strpos($token, 'alt_text&quot') === FALSE) {
```

Comment:
So this is a defined standard somewhere?

Reply:
https://git.drupalcode.org/project/entity_embed/blob/8.x-1.x/src/Plugin/entity_embed/EntityEmbedDisplay/ImageFieldFormatter.php#L186